tests/test_ind_indygo_service.py

```diff
@@ -40,29 +40,26 @@ def test_description():
 
 
 def test_start():
-    assert parsed_items[0]["start"] == datetime(2023, 1, 19, 10, 0)
+    assert parsed_items[0]["start"] == datetime(2024, 1, 18, 10, 0)
 
 
 # def test_end():
 #     assert parsed_items[0]["end"] == datetime(2019, 1, 1, 0, 0)
 
 
 def test_time_notes():
-    assert (
-        parsed_items[0]["time_notes"]
-        == "Service Committee meetings are set for 10:00AM unless otherwise noted in meeting description. Please double check the website before the meeting date."  # noqa
-    )  # noqa
+    assert parsed_items[0]["time_notes"] == ""  # noqa  # noqa
```

Comment:
We seem to have two `# noqa`s here so we should remove one, at the very least. Ideally, though, we should try resolve the linting error rather than suppressing it. I would recommend removing both `# noqa`s and running `flake8 .` from the pipenv shell to get a breakdown of the linting errors and then fixing those errors.

For clean code, I would repeat this in any other changed files where we may have used `# noqa` a bit too liberally. Using `# noqa for long lines is fine though 🙂

Reply:
@SimmonsRitchie Roger that.